Tasks/SonarQubePostTest/SonarQubeMetrics.ps1

```diff
@@ -55,7 +55,7 @@ function FetchQualityGateDetails
 #
 function FetchMetricNames
 {
-    $response = InvokeGetRestMethod "/api/metrics/search?ps=500&f=name" $false
+    $response = InvokeGetRestMethod "/api/metrics/search?ps=500&f=name" $true
```

Comment:
Part of testing I've discovered this API is improperly documented and it actually requires authentication. I am discussing with SonarSource what should be done further, but at the very least I'll be passing credentials here.